### DISCUSSION TITLE
Make the bases smaller

### DISCUSSION THREAD

**Discussion by Terri-Lea** [fjossinet/RNArtist#26] [Q&A]
I'd like to see the letters for the base codes but sometimes due to the structure they get squished and can no longer be seen. How do I make the residues smaller but still visible?

**Comment by fjossinet** [comment]
Hi Terri-Lea,

Sorry for this late reply. Do you mean this kind of situation (bases 40 to 45):

<img width="237" alt="Capture d’écran 2023-12-06 à 09 27 08" src="https://github.com/fjossinet/RNArtist/assets/389452/1121b8e7-eeab-4d85-97e5-5fcccfe65263">

You could select this junction (left click on it) and change its size and layout with the layout panel on the right:

<img width="582" alt="Capture d’écran 2023-12-06 à 09 29 32" src="https://github.com/fjossinet/RNArtist/assets/389452/63fde55d-da53-4569-b4be-be36b1246584">

The up/down arrows change the junction size. Click on an orange circle to remove it and click on another empty circle to define the new location for an helix.

<img width="932" alt="Capture d’écran 2023-12-06 à 09 30 05" src="https://github.com/fjossinet/RNArtist/assets/389452/03122e40-40d6-48d7-99a7-8d3b8e62217c">

Does it help?
